Documentation/PageTsconfig/TceForm.rst

```diff
@@ -293,13 +293,13 @@ config
    This property is available for various levels:
 
    table level, example:
-      `TCEFORM.tt_content.header.config.max`
+      `TCEFORM.tt_content.header.config.maxitems`
```

Comment:
You have to distinguish between maxItems (= select items) and max characters (in input field). Have you looked into the code=